requirements.txt

```diff
@@ -1 +1,2 @@
-ops >= 2.2.0
+ops==2.12.0
```

Comment:
Can you include a reference to why we're pinning to this specific version of ops?

Reply:
All dependencies should be pinned up to the patch version according to our specs

Reply:
Ah right, I'd misread this as being a lower version than what it was set to previously. I realise now this is the latest released version, so sounds good to me.